app/controllers/bulk_actions/descriptive_metadata_import_jobs_controller.rb

```diff
@@ -7,7 +7,7 @@ class DescriptiveMetadataImportJobsController < ApplicationController
 
     def create
       csv = CSV.parse(normalized_csv, headers: true)
-      validator = DescriptionValidator.new(csv)
+      validator = DescriptionValidator.new(csv, bulk_job: true)
       if validator.valid?
```

Comment:
the "druid" column is only present in bulk jobs CSVs, this allows us to send a signal to validation to do this extra check